tests/backup/backup_helper.go

```diff
@@ -96,7 +96,7 @@ const (
 	backupLocationDeleteRetryTime             = 30 * time.Second
 	rebootNodeTimeout                         = 1 * time.Minute
 	rebootNodeTimeBeforeRetry                 = 5 * time.Second
-	latestPxBackupVersion                     = "2.4.0"
+	latestPxBackupVersion                     = "2.6.0"
```

Comment:
are these params not parameterized

Reply:
These are parameterized. In case we miss providing the env variable it will fall back to this.